examples/docker/Makefile

```diff
@@ -1,8 +1,8 @@
 all: binaries build up
 
 binaries:
-	GOOS=linux go build -o ca/step-ca github.com/smallstep/certificates/cmd/step-ca
```

Comment:
@maraino did we purposely remove the `CGO_ENABLED=0` vars at some point. I could be remembering a different instance. Just want to make sure we don't add something we purposefully removed.

Reply:
We didn't remove it, it was never added, and it wasn't causing a problem because YubiKey support wasn't there.